Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
```

Comment:
You can consider adding labels to the image, for example about the authors/license etc. For more info see here: https://github.com/opencontainers/image-spec/blob/master/annotations.md

For example, to point users to the documentation: 

LABEL org.opencontainers.image.documentation="https://github.com/SuperElastix/elastix/wiki"